scripts/check_public_api_docstrings.py

```diff
@@ -0,0 +1,189 @@
+import logging
+import pathlib
+import re
+import subprocess
+from dataclasses import dataclass
+from typing import List, Set, Tuple
+
+from scripts.public_api_report import (
+    FileContents,
+    _default_code_absolute_paths,
+    CodeParser,
+    PublicAPIChecker,
+    Definition,
+)
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
logging.basicConfig(level=logging.INFO)
```
I would use `basicConfig()` here.
https://docs.python.org/3/library/logging.html#logging.basicConfig